src/main/java/com/exacttarget/fuelsdk/ETClient.java

```diff
@@ -154,11 +157,13 @@ public ETClient(ETConfiguration configuration)
             gson = gsonBuilder.create();
         }
 
+        useOAuth2Authentication = configuration.isTrue("useOAuth2Authentication") ? true : false;
```

Comment:
does this error out when the property is not available in config file?

Reply:
No. There is a null check for the case when the key is missing from the config file and the method returns false.